docs/ci.md

```diff
@@ -0,0 +1,39 @@
+# Continuous Integration
```

Comment:
nit: add this `Continuous Integration (CI)` as title so CI acronym is explained already.

Reply:
done